test/gleam/http_test.gleam

```diff
@@ -1111,3 +1111,26 @@ pub fn expire_resp_cookie_test() {
     "k1=; Expires=Thu, 01 Jan 1970 00:00:00 GMT; Max-Age=0; Path=/; HttpOnly",
   ))
 }
+
+pub fn get_req_origin_test() {
+  let origin = "http://example.com"
+  let referer = "http://test.com/elixir"
```

Comment:
It might be worth making this reflect reality and have the referrer as `http://example.com/elixir`. Origin and referrer would not be different. 

Probably not that importat

Reply:
That makes sense, thanks. Updated to use the proper host in the referer